Cds 92419 (#6700)
harness/developer-hub: docs/platform/git-experience/import-input-sets.md

This topic shows you how to import an Input Set to the CD module.

> 1. Select your Project and click on **Pipelines** and select a pipeline.

Reviewer: ```suggestion
1. Select your project, select **Pipelines**, then select a pipeline.
```
